src/groups/meta_group_wrapper.cpp

```diff
@@ -9,6 +9,68 @@
 
 namespace session::nodeapi {
 
+Napi::Object member_to_js(const Napi::Env& env, const member& info, const member::Status& status) {
+    auto obj = Napi::Object::New(env);
+
+    obj["pubkeyHex"] = toJs(env, info.session_id);
+    obj["name"] = toJs(env, info.name);
+    obj["profilePicture"] = toJs(env, info.profile_picture);
+
+    switch (status) {
+        // invite statuses
+        case member::Status::invite_unknown:
+            obj["memberStatus"] = toJs(env, "INVITE_UNKNOWN");
+            break;
+        case member::Status::invite_not_sent:
+            obj["memberStatus"] = toJs(env, "INVITE_NOT_SENT");
+            break;
+        case member::Status::invite_sending:
+            obj["memberStatus"] = toJs(env, "INVITE_SENDING");
+            break;
+        case member::Status::invite_failed: obj["memberStatus"] = toJs(env, "INVITE_FAILED"); break;
+        case member::Status::invite_sent: obj["memberStatus"] = toJs(env, "INVITE_SENT"); break;
+        case member::Status::invite_accepted:
+            obj["memberStatus"] = toJs(env, "INVITE_ACCEPTED");
+            break;
+
+        // promotion statuses
+        case member::Status::promotion_unknown:
+            obj["memberStatus"] = toJs(env, "PROMOTION_UNKNOWN");
+            break;
+        case member::Status::promotion_not_sent:
+            obj["memberStatus"] = toJs(env, "PROMOTION_NOT_SENT");
+            break;
+        case member::Status::promotion_sending:
+            obj["memberStatus"] = toJs(env, "PROMOTION_SENDING");
+            break;
+        case member::Status::promotion_failed:
+            obj["memberStatus"] = toJs(env, "PROMOTION_FAILED");
+            break;
+        case member::Status::promotion_sent:
+            obj["memberStatus"] = toJs(env, "PROMOTION_SENT");
+            break;
+        case member::Status::promotion_accepted:
+            obj["memberStatus"] = toJs(env, "PROMOTION_ACCEPTED");
+            break;
+
+        // removed statuses
+        case member::Status::removed_unknown:
+            obj["memberStatus"] = toJs(env, "REMOVED_UNKNOWN");
+            break;
+        case member::Status::removed: obj["memberStatus"] = toJs(env, "REMOVED_MEMBER"); break;
```

Comment:
Should the break not be on a new line?

Reply:
This might just be an autoformat thing. If so you can ignore and resolve these comments